src/main/webapp/creatives/maps/UserActivityLogger.js

```diff
@@ -13,33 +14,33 @@ class UserActivityLogger {
 
   /**
    * Makes a GET request to the given URL with the correlator as a parameter.
-   * @param {string} url A URL to send the request to.
+   * @param {string} path A URL to send the request to.
    */
-  sendRequest_(url) {
-    fetch(`/${url}?correlator="${this.correlator_}"`);
+  sendRequest_(path) {
```

Comment:
You & Julia have different things written under this function-- do you need to include the mode: 'no-cors' here?

Reply:
we tested it oiut and it wasn't necessary